descriptors/ModuleDescriptor-template.json

```diff
@@ -899,6 +918,11 @@
       "displayName": "circulation - request queue for an item",
       "description": "get request queue for an item"
     },
+    {
+      "permissionName": "circulation.requests.report.get",
```

Comment:
This name suggests a single permission for any report. Please could it be changed to be more specific to this report

Reply:
done